internal/zinc-core/src/main/scala/sbt/internal/inc/APIDiff.scala

```diff
@@ -57,29 +30,202 @@ private[inc] class APIDiff {
                       contextSize: Int): String = {
     val api1Str = DefaultShowAPI(api1.classApi) + "\n" + DefaultShowAPI(api1.objectApi)
     val api2Str = DefaultShowAPI(api2.classApi) + "\n" + DefaultShowAPI(api2.objectApi)
-    generateApiDiff(fileName, api1Str, api2Str, contextSize)
+    DiffUtil.mkColoredCodeDiff(api1Str, api2Str, printDiffDel = true)
   }
 
-  private def generateApiDiff(fileName: String, f1: String, f2: String, contextSize: Int): String = {
-    assert((diffMethod != null) && (generateUnifiedDiffMethod != null),
-           "APIDiff isn't properly initialized.")
-    import scala.collection.JavaConverters._
-    def asJavaList[T](it: Iterator[T]): java.util.List[T] = it.toSeq.asJava
-    val f1Lines = asJavaList(f1.lines)
-    val f2Lines = asJavaList(f2.lines)
-    //val diff = DiffUtils.diff(f1Lines, f2Lines)
-    val diff /*: Patch*/ = diffMethod.invoke(null, f1Lines, f2Lines)
-    val unifiedPatch: JList[String] = generateUnifiedDiffMethod
-      .invoke(null, fileName, fileName, f1Lines, diff, (contextSize: java.lang.Integer))
-      .asInstanceOf[JList[String]]
-    unifiedPatch.asScala.mkString("\n")
-  }
+  /**
+   * This class was directly copied from Dotty [[https://github.com/lampepfl/dotty/blob/master/compiler/src/dotty/tools/dotc/util/DiffUtil.scala]]
+   */
+  private object DiffUtil {
```

Comment:
FTR: This implementation does not depend on java-diff-utils, that's why it can be removed.